flask_classful.py

```diff
@@ -422,7 +423,8 @@ def get_interesting_members(base_class, cls):
             )
             and not member[0].startswith("_")
             and not member[0].startswith("before_")
-            and not member[0].startswith("after_")]
+            and not member[0].startswith("after_")
```

Comment:
This list comprehension is getting kinda crazy -- recommend re-writing as a regular loop...

Reply:
thank you, we'll consider this later.